confirm-deposit

```diff
@@ -1 +1,109 @@
-# Title
+# Confirm a Deposit to the Exchange
+
+Deposits are made in several stages. 
+One of the stages is to go through the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange), which monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a blockchain, multiple [validators](h[ttps://docs.flare.network/tech/glossary/](https://docs.flare.network/tech/validators/)) examine newly created transactions and come to an agreement about whether they are valid. 
+When they are valid, they are added to a new block on the chain and identified by a one-way cryptographic algorithm, called a _hash_. 
+The one-way hash prevents transactions from being reverted. 
+[Blocks](https://docs.flare.network/tech/glossary/) are added to the [blockchain](https://docs.flare.network/tech/glossary/) in chronological order and it may take a few blocks for the hash to be completed.
+To prevent problems, we confirm deposits several blocks back (five blocks in our code), when the blocks are most likely to be irreversible.
+
+Newly submitted deposits to our receiving address are labeled "pending."
+After five blocks have been created, valid deposits are labeled "confirmed."
+
+Run the javascript below to see pending and confirmed transactions and which block number they were added to.
+
+## One-Time Setup
+
+1. Install [node.js](https://nodejs.org/en/download/). 
+On the command line, run `node -v`  to check your installation.
+
+2. Clone the Flare repository: 
+
+`git clone https://github.com/flare-foundation/flare.git
+cd flare`
+
+See [the Flare repository on GitHub](https://github.com/flare-foundation/flare).
+
+3. Install the web3.js library: 
+
+`npm install web3`
+
+See [Adding web3.js](https://web3js.readthedocs.io).
+
+4. Build Flare with this script:
+
+`./scripts/build.sh`
+
+## Run the Code
+
```

Comment:
How do they run this file? I've seen people run files in the cli, but I don't know how to do it. I created a file and ran `npm run confirm-deposit.js`, but it didn't work.

Reply:
It should work with `node confirm-deposit.js`.
I am no expert on node or js, so I am not sure what `npm run...` does.
In the future, whenever something does not work, give a bit more of information (what you typed and what the output was, for example) :)

Reply:
Mistake, I think I meant `node run confirm-deposit.js` and I may have done it before I downloaded web3. So I'll try again.

Reply:
:+1: